lib/netsuite/version.rb

```diff
@@ -1,3 +1,3 @@
 module Netsuite
-  VERSION = '0.7.1'
+  VERSION = '0.7.1.2'
```

Comment:
@jeperkins4 Can you remove the version bump? I'll handle this when I merge the branch.